.eleventy.js

```diff
@@ -47,11 +47,14 @@ module.exports = (eleventyConfig) => {
       .filter((item) => item.data.bird)
       .sort((a, b) => a.date - b.date),
   );
+  eleventyConfig.addCollection('_sorted-posts', (collection) =>
+    pages.eventSort(collection.getFilteredByTag('_post')),
+  );
   eleventyConfig.addCollection('oss', (collection) =>
     collection
       .getAll()
       .filter((item) => item.data.oss)
-      .sort((a, b) => b.date - a.date),
+      .sort((a, b) => b.sort - a.sort),
```

Comment:
Oh, right - We'd have to run them through our `eventSort` or `pageYears` filter, which adds that property…

Reply:
Fixed -- though maybe we want a manual sort for OSS projects instead of most-recent-first? That would allow us to highlight the bigger or more active ones.

Reply:
oh yeah, didn't see this was on OSS. I think we have a card for that already, just got de-prioritized: https://trello.com/c/dGgqSrQ9/58-i-want-featured-oss-projects